azerothcore/azerothcore-wotlk
# fix(Core/Wintergrasp): Spam msg's

Wintergrasp - Spam msg's (WIP - for any other improvement. But can be tested.)

##### CHANGES PROPOSED:

-  Correctly sorted with text's
-  


###### ISSUES ADDRESSED:


Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1869


##### TESTS PERFORMED:

Didn't test everything. Still working on this


##### HOW TO TEST THE CHANGES:

1. Play Wintergrasp and check Chat log / Attack, Capture bases, Capture Relic etc..
2. Check will you get same spam, like on this image : 

![Winter](https://user-images.githubusercontent.com/41213210/60548258-f2882100-9d21-11e9-91ae-26a470ba9d01.jpg)



##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
